api/v1alpha1/flowcollector_types.go

```diff
@@ -194,6 +207,19 @@ type FlowCollectorConsolePlugin struct {
 	Resources corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,8,opt,name=resources"`
 }
 
+// CNO defines the desired configuration related to the Cluster Network Configuration
+type ClusterNetworkOperator struct {
+	// Important: Run "make generate" to regenerate code after modifying this file
+
+	//+kubebuilder:default:=ovs-flows-config
+	// Namespace  where the configmap is going to be deployed.
+	ConfigmapName string `json:"configmapName,omitempty"`
```

Comment:
Here also, not sure if there's a real need to configure the config map name that we generate (moreover if it's configurable here then it must also be configurable in the CNO resource, not sure if that's the case, @mariomac is that something you did?)

Reply:
I don't understand, was this constant referred in another repo ?

Reply:
yes this config map is later watched by the CNO, see this story: https://issues.redhat.com/browse/NETOBSERV-66

Reply:
so i guess the name matters here

Reply:
Yeah I did this... maybe I was being too configurator. Feel free to remove it if you think it's better to just put a constant.

Reply:
What I mean is, here https://github.com/openshift/cluster-network-operator/pull/1231/files#diff-993ef112f8af6a1b2aaf1b7f141a10306d0420db5b56a272d7365a8bacd82920R25 it's hardcoded, so if it's hardcoded on one side, there's no need to make it configurable on the other end, right?
It could give a wrong impression that the user can set whatever he wants, but in reality changing this would break on the CNO side